inst/manuscript/manuscript.Rmd

```diff
@@ -334,9 +334,11 @@ Users can obtain PIT histograms based on validated forecast objects using the fu
 ```{r pit-plots, fig.pos = "!h", fig.cap="PIT histograms of all models stratified by forecast target. Histograms should ideally be uniform. A u-shape usually indicates overconfidence (forecasts are too narrow), a hump-shaped form indicates underconfidence (forecasts are too uncertain) and a triangle-shape indicates bias.", fig.width = 8, fig.height=4}
 example_sample_continuous |>
   as_forecast_sample() |>
-  get_pit(by = c("model", "target_type")) |>
-  plot_pit() + 
-  facet_grid(target_type ~ model)
+  get_pit_histogram(by = c("model", "target_type")) |>
+  ggplot(aes(x = mid, y = density)) +
```

Comment:
I'd personally prefer we do ship a simpler wrapper around this code (and the labs) to reduce duplication though obviously it isn't very complicated

Reply:
Do you mean to reinstate `plot_pit()`, maybe even as S3 function that calls `get_pit_histogram()` and then plots? Or something else?